packages/react/etc/components-react.api.md

```diff
@@ -695,6 +710,20 @@ export function useSortedParticipants(participants: Array<Participant>): Partici
 // @public
 export const useSpeakingParticipants: () => Participant[];
 
+// @alpha
+export function useStartAudio({ room, props }: UseStartAudioProps): {

```

Comment:
same here, moving `ensureRoom` from the component to the hook seems sensible to me